pages/add-clinic.vue

```diff
@@ -5,65 +5,87 @@
       v-model="valid"
       lazy-validation
       @submit.prevent="submitData"
+      class="mb-8 mt-8"
```

Comment:
Can be done in separate PR:

We should make a class Anne relevant to the actual element the. Apply these classes in a sass rule.